lib/private/remove-non-userland-calls-from-stack.js

```diff
@@ -55,11 +58,10 @@ module.exports = function removeNonUserlandCallsFromStack(stackEntries) {
       // Trim whitespace from the path and use path.dirname to get its directory.
       var fileDir = path.dirname(filePath.trim());
```

Comment:
for future reference: `String.prototype.trim()` is not supported in IE8.  That's not a concern for us right now (like the client-side part of the XSS stuff I did in Sails, we should only worry about >= IE9 in our browserified stuff for the time being) -- just making sure there's a note about it somewhere.

(See http://kangax.github.io/compat-table/es5/#test-String_properties_and_methods_String.prototype.trim)